snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -95,15 +95,15 @@ public void CreateBucket()
         {
             var projectId = _fixture.ProjectId;
 
-            // <Buckets.Insert>
+            // <Service.Buckets.Insert>
```

Comment:
Don't worry about this for the moment - basically the snippets are only likely to be meaningful within StorageClient. But might as well leave the commit in place.